frontend/src/assets/News/news.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
this is the text on the home page of the application in the different languages. this will change from time to time ... and this is what i did not want to have in the locales